examples/c-hello-world/nixel/builders.ncl

```diff
@@ -90,7 +90,6 @@ let {NickelDerivation, ..} = import "contracts.ncl" in
 
     output.structured_env = {
       PATH = {
-        bash = nix-s%"%{inputs.bash}/bin"%,
```

Comment:
This change doesn't seem to be related to the Nickel version. What is the reason to remove bash?

Reply:
This file was out of sync with one in the root. I’ve just copied the file from root here.